pom.xml

```diff
@@ -17,6 +17,10 @@
             <url>http://opensource.org/licenses/MIT</url>
         </license>
     </licenses>
+    <properties>
+        <jenkins.version>1.580.1</jenkins.version>
+        <jenkins-test-harness.version>1.580.1</jenkins-test-harness.version>
```

Comment:
In other plugins conversions the same pull request has included the,  update the parent POM,  test fixes so that they compile with JTH 2.1 and javadoc fixes so that it does the build does not break on Java 8 If the changes are not too big, encourages adoption.